src/Compression/CompressionCodecQATZSTD.cpp

```diff
@@ -0,0 +1,114 @@
+#ifdef ENABLE_QATZSTD_COMPRESSION
```

Comment:
Docs in docs/en/sql-reference/statements/create/table.md are needed.

Reply:
Done